dictionary/README.md

```diff
@@ -2,4 +2,6 @@
 Please write a couple of sentences explaining your choice of restart
 strategy.
 
-
+The best restart strategy is :transient since there is no need to restart the
```

Comment:
# -2

I see your logic, but the dictionary should never exit normally. As a result, :transient and :permanent are effectively the same, but :permanent expresses intent way better.